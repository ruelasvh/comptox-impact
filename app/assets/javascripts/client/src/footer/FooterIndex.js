/**
 * Created by Victor H. Ruelas-Rivera on 1/27/17.
 * US EPA National Center for Computational Toxicology
 */
import React from 'react';
import seal from './img/epa_logo_round.svg';
import blog from './img/socmed_blog.png';
import twitter from './img/socmed_twitter_2.png';
import facebook from './img/socmed_facebook_2.png';
import youtube from './img/socmed_youtube_2.png';
import news from './img/socmed_news.png';
import './FooterIndex.css';

const FooterIndex = () => (
    <footer className="footer">
        <div className="footer-logo">
            <a href="https://epa.gov">
                <img src={seal} alt="EPA_Seal"/>
            </a>
        </div>

        <ul className="footer-follow-links">
            <li><a href="https://www.youtube.com/user/USEPAgov" target="_blank"><img src={youtube} alt="socmed_youtube"/></a></li>
            <li><a href="https://www.facebook.com/EPAresearch/" target="_blank"><img src={facebook} alt="socmed_facebook"/></a></li>
            <li><a href="https://www.twitter.com/epa" target="_blank"><img src={twitter} alt="socmed_twitter"/></a></li>
            <li><a href="https://blog.epa.gov/blog/tag/chemical-research/" target="_blank"><img src={blog} alt="socmed_blog"/></a></li>
            <li><a href="https://www.epa.gov/newsreleases/search/rss/subject/research" target="_blank"><img src={news} alt="socmed_news"/></a></li>
        </ul>

        <ul className="footer-links">
            <li><div><a href="/">Home</a></div></li>
            <li><div><a href="https://epa.gov/comptox" target="_blank">About Us</a></div></li>
            <li><div><a href="#PrivacySecurityNotice">Privacy and Security Notice</a></div></li>
            <li><div><a href="#Accesibility">Accessibility</a></div></li>
            <li><p style={{fontSize: "small"}}>Last Updated on {document.lastModified}</p></li>
        </ul>
    </footer>
);

export default FooterIndex;