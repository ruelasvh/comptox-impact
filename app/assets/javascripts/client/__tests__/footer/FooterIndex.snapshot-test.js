/**
 * Created by Victor H. Ruelas-Rivera on 1/31/17.
 * US EPA National Center for Computational Toxicology
 */
import React from 'react';
import FooterIndex from '../../src/footer/FooterIndex';
import renderer from 'react-test-renderer';

test('FooterIndex renders correctly', () => {
    const component = renderer.create(
        <FooterIndex />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});