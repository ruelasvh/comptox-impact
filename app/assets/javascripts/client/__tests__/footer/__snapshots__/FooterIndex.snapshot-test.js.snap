exports[`test FooterIndex renders correctly 1`] = `
<footer
  className="footer">
  <div
    className="footer-logo">
    <a
      href="https://epa.gov"
      target="_blank">
      <img
        alt="EPA_Seal"
        src="test-file-stub" />
    </a>
  </div>
  <ul
    className="footer-follow-links">
    <li>
      <a
        href="https://www.youtube.com/user/USEPAgov"
        target="_blank">
        <img
          alt="socmed_youtube"
          src="test-file-stub" />
      </a>
    </li>
    <li>
      <a
        href="https://www.facebook.com/EPAresearch/"
        target="_blank">
        <img
          alt="socmed_facebook"
          src="test-file-stub" />
      </a>
    </li>
    <li>
      <a
        href="https://www.twitter.com/epa"
        target="_blank">
        <img
          alt="socmed_twitter"
          src="test-file-stub" />
      </a>
    </li>
    <li>
      <a
        href="https://blog.epa.gov/blog/tag/chemical-research/"
        target="_blank">
        <img
          alt="socmed_blog"
          src="test-file-stub" />
      </a>
    </li>
    <li>
      <a
        href="https://www.epa.gov/newsreleases/search/rss/subject/research"
        target="_blank">
        <img
          alt="socmed_news"
          src="test-file-stub" />
      </a>
    </li>
  </ul>
  <ul
    className="footer-links">
    <li>
      <div>
        <a
          href="/">
          Home
        </a>
      </div>
    </li>
    <li>
      <div>
        <a
          href="https://epa.gov/comptox"
          target="_blank">
          About Us
        </a>
      </div>
    </li>
    <li>
      <div>
        <a
          href="#PrivacySecurityNotice">
          Privacy and Security Notice
        </a>
      </div>
    </li>
    <li>
      <div>
        <a
          href="#Accesibility">
          Accessibility
        </a>
      </div>
    </li>
    <li>
      <p
        style={
          Object {
            "fontSize": "small",
          }
        }>
        Last Updated on 
        03/29/2017 11:28:23
      </p>
    </li>
  </ul>
</footer>
`;
